locksmith/src/worker/tasks/checkoutHooks.ts

```diff
@@ -0,0 +1,38 @@
+import { Task } from 'graphile-worker'
+import { Payload } from '../../models/payload'
+import logger from '../../logger'
+import axios from 'axios'
+import { getCheckoutConfigById } from '../../operations/checkoutConfigOperations'
+
+export const checkoutHookJob: Task = async (payload: any) => {
```

Comment:
I don't think we should send these from the backend to be honest. They should be triggered from the front-end directly